src/app/index.tsx

```diff
@@ -51,6 +52,7 @@ export function App() {
               <Route exact path="/vote" component={VotePage} />
               <Route exact path="/trade" component={TradePage} />
               <Route exact path="/airdrip" component={Airdrip} />
+              <Route path="/proposal/:id" component={ProposalPage} />
```

Comment:
change it `path = "/vote/proposal/:id"`
If a user navigates to see a proposal, the `Vote` tab would be active. React-Router-Dom automatically detects the active path if you give the path like the above.